README.lhs

```diff
@@ -366,7 +361,7 @@ instance Yesod App where
   -- ...
 
   messageLoggerSource app _logger loc source level msg =
-    runLoggerLoggingT app $ monadLoggerLog loc source level msg
```

Comment:
It seems that these integration instructions were doing it wrong, since this runs on a per-request rather than per-application level.

Reply:
Yeah, totally. It's great that we can avoid it now. I think we (or anyone) were just lucky because you very rarely actually hit this hook function in a Yesod app. I honestly don't even know when it's used.